src/pages/Bridge/BridgeOverview/components/IssueForm/IssueForm.tsx

```diff
@@ -154,6 +164,21 @@ const IssueForm = ({ requestLimits, dustValue, issueFee }: IssueFormProps): JSX.
     }
   });
 
+  const griefingCollateralTicker = form.values[BRIDGE_ISSUE_GRIEFING_COLLATERAL_TOKEN];
+
+  useEffect(() => {
+    const computeSecurityDeposit = async () => {
+      const btcAmount = safeBitcoinAmount(amount || 0);
+      const deposit = await getSecurityDeposit(btcAmount, griefingCollateralTicker);
+
+      if (!deposit) return;
+
+      setSecurityDeposit(deposit);
+    };
+
+    computeSecurityDeposit();
+  }, [amount, griefingCollateralTicker, setSecurityDeposit, getSecurityDeposit]);
```

Comment:
@peterslany I replaced `form.values` with `griefingCollateralTicker` because the useEffect does not need to re-run on each form values changes.